lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -475,9 +475,9 @@ dictionary Payment {
     u64 fees_sat;
     PaymentType payment_type;
     PaymentState status;
+    PaymentDetails details;
```

Comment:
@hydra-yse should we also make the destination field now mandatory as well?

Reply:
@roeierez Looking at the code, I think that requires a PR of its own. It can be done, but we need to set the `bolt11` field in the details as mandatory, as well as the `claim_address` field for chain swaps (since they're now optional and we use them to rebuild the destination)